docs/source/cql.rst

```diff
@@ -48,3 +56,41 @@ A ``between`` example for a specific property through an HTTP POST request:
       "upper": 0.60
     }
   }'
+
+Or 
+
+.. code-block:: bash
+
+  curl --location --request POST 'https://ogcapi.bgs.ac.uk/collections/recentearthquakes/items?f=json&limit=10&filter-lang=cql-json' 
+  --header 'Content-Type: application/query-cql-json' 
+  --data-raw '{ 
+    "between":{
+      "value":{"property": "ml"},
+      "lower":4,
+      "upper":4.5
+    }
+  }'
+
+The same ``BETWEEN`` query using HTTP GET request formatted as CQL text and URL encoded as below:
+
+.. code-block:: bash
+
+ curl "https://ogcapi.bgs.ac.uk/collections/recentearthquakes/items?f=json&limit=10&filter=ml%20BETWEEN%204%20AND%204.5"
+
+An ``EQUALS`` example for a specific property:
+
+.. code-block:: bash
+  curl --location --request POST 'https://ogcapi.bgs.ac.uk/collections/recentearthquakes/items?f=json&limit=10&filter-lang=cql-json' 
+  --header 'Content-Type: application/query-cql-json' 
+  --data-raw '{
+      "eq":[{"property": "user_entered"},"APBE"]
+  }'
+
+A ``CROSSES`` example via an HTTP GET request.  The CQL text is passed via the ``filter`` parameter.
+
+.. code-block:: bash
+
+  curl "http://localhost:5000/collections/hot_osm_waterways/items?f=json&filter=CROSSES(foo_geom,%20LINESTRING(28%20-2,%2030%20-4))"
+
+Note the the CQL text has been URL encoded here.
```

Comment:
Note that the CQL text has been URL encoded.

Reply:
Thats the current text in the docs - we've not changed that

Reply:
Fixed in latest commits by @ximenesuk and @KoalaGeo.